src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -166,8 +181,42 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
     }
 
     /**
-     * This method saves the missed Notification into a single document along with
-     * Subscription name, notification meta and time period.
+     * This method reformats notificationMeta and fetches values if applicable
+     * for the requested parameters.
+     *
+     * @param aggregatedObject
+     * @param notificationMeta
+     * @return String
+     */
+    private String reformatNotificationMeta(String aggregatedObject, String notificationMeta) {
+        String URL = notificationMeta.split("\\?")[0];
+
+        List<NameValuePair> params = null;
+        try {
+            params = URLEncodedUtils.parse(new URI(notificationMeta), Charset.forName("UTF-8"));
+        } catch (Exception e) {
+            LOGGER.error(e.getMessage(), e);
```

Comment:
Can you add some short description on what failed before exception message in the log message.

Reply:
Done.